src/core/Wizard/Step.test.tsx

```diff
@@ -6,384 +6,195 @@ import { render } from '@testing-library/react';
+        index={1}
+        currentStepNumber={0}
+        totalSteps={4}
+        type={type}
+      />,
     );
 
-    const { container } = render(step);
-
-    // Renders as active
-    expect(container.querySelector('.iui-wizards-step-current')).toBeTruthy();
-    // No track before
-    expect(
-      container.querySelector('.iui-wizards-step-track-before'),
-    ).toBeNull();
-    // No title
-    expect(container.querySelector('iui-wizards-step-title')).toBeNull();
-    // Track after
-    expect(container.querySelector('.iui-wizards-step-track-after')).toBeNull();
-    // Circle
-    const circle = container.querySelector(
-      '.iui-wizards-step-circle',
-    ) as HTMLElement;
-    expect(circle).toBeTruthy();
-    expect(circle.textContent).toBe('Last step');
-    // Main track
-    expect(container.querySelector('.iui-wizards-step-track-main')).toBeNull();
+    const step = container.querySelector('.iui-wizard-step') as HTMLElement;
+    expect(step).toBeTruthy();
+    expect(step.style.width).toBeFalsy(); // not 25%
   });
 });
 
-it('should set dynamic inline width based on total steps', () => {
-  const { container: container1 } = render(
-    <Step
-      title='Mock step'
-      index={1}
-      currentStepNumber={0}
-      totalSteps={4}
-      type={'default'}
-    />,
-  );
-
-  const step1 = container1.querySelector('.iui-wizards-step') as HTMLElement;
-  expect(step1).toBeTruthy();
-  expect(step1.style.width).toEqual('25%');
-
-  const { container: container2 } = render(
-    <Step
-      title='Mock step'
-      index={1}
-      currentStepNumber={1}
-      totalSteps={8}
-      type={'default'}
-    />,
-  );
-
-  const step2 = container2.querySelector('.iui-wizards-step') as HTMLElement;
-  expect(step2).toBeTruthy();
-  expect(step2.style.width).toEqual('12.5%');
-});
-
-it('should not set dynamic inline width for long and workflow wizard types', () => {
-  const { container: container1 } = render(
-    <Step
-      title='Mock step'
-      index={1}
-      currentStepNumber={0}
-      totalSteps={4}
-      type={'long'}
-    />,
-  );
-
-  const step1 = container1.querySelector('.iui-wizards-step') as HTMLElement;
-  expect(step1).toBeTruthy();
-  expect(step1.style.width).toBeFalsy(); // not 25%
-
-  const { container: container2 } = render(
-    <Step
-      title='Mock step'
-      index={1}
-      currentStepNumber={1}
-      totalSteps={8}
-      type={'workflow'}
-    />,
-  );
-
-  const step2 = container2.querySelector('.iui-wizards-step') as HTMLElement;
-  expect(step2).toBeTruthy();
-  expect(step2.style.width).toBeFalsy(); // not 12.5%
-});
-
-it('should add className and style props correctly', () => {
-  const { container } = render(
-    <Step
-      title='Mock step'
-      index={1}
-      currentStepNumber={0}
-      totalSteps={4}
-      type={'default'}
-      className='custom-class'
-      style={{ width: 50, color: 'red' }}
-    />,
-  );
+describe('Wizard step', () => {
```

Comment:
dont understand sections..
There is default wizard step and there is wizard step...

Reply:
The difference is that this section is generic and the test would be the same across all types.

I guess it does not need to be inside `describe`.

Reply:
Removed